cli/main.go

```diff
@@ -266,6 +271,17 @@ func beforeSubcommands(c *cli.Context) error {
 		return nil
 	}
 
+	r, err := parseBoolOrAuto(c.GlobalString("rootless"))
+	if err != nil {
+		return err
+	}
+	// If nil, not set a value, let vc auto-detect rootless
```

Comment:
This comment is slightly confusing, as we are checking non-nil value below. Perhaps, this could be worded as
`If flag is true/false, assign the rootless flag. vc will not perform any auto-detection in that case. In case flag is nil or auto, vc detects if the runtime is running as rootless.`

Reply:
it's set in `TestMain` (entrypoint) https://github.com/kata-containers/runtime/pull/2429/files#diff-e8e7e93733a02e75346a2f9d43ad6149R113